sha256-hasher/src/lib.rs

```diff
@@ -1,15 +1,15 @@
 #![no_std]
```

Comment:
I can't comment on the specific [lines](https://github.com/anza-xyz/solana-sdk/pull/236/files#diff-be8b223a57abdbd0bcf2507146cb5d6e570ed443f52b1911679133a8c95803d4R69-R74), but this files also has an `extend_and_hash` implementation, which can be removed - same behaviour can be achieved with the `hashv`.